src/mlpack/methods/naive_bayes/naive_bayes_classifier.hpp

```diff
@@ -43,11 +43,20 @@ namespace naive_bayes /** The Naive Bayes Classifier. */ {
  *
  * nbc.Classify(testing_data, results);
  * @endcode
+ *
+ * The ModelMatType template parameter specifies the internal matrix type that
+ * NaiveBayesClassifier will use to hold the model.  This can be arma::mat,
+ * arma::fmat, or any other Armadillo (or Armadillo-compatible) object.
```

Comment:
hm, I think we should make it clear why we do this here, the PR summary sounds for me more informative.